docker-compose.yml

```diff
@@ -13,7 +13,6 @@
  # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  # See the License for the specific language governing permissions and
  # limitations under the License.
-version: "3.3"
```

Comment:
fix docker winning: ../certification-tool/docker-compose.yml: `version` is obsolete